.buildkite/pipeline.yml

```diff
@@ -92,6 +96,13 @@ steps:
           slurm_mem_per_cpu: 8G
           slurm_gpus: 1
 
+      - label: "Global Bucket on GPU"
+        command: "julia --color=yes --project=experiments experiments/standalone/Bucket/global_bucket.jl"
+        agents:
+          slurm_mem_per_cpu: 8G
+          slurm_gpus: 1
+        artifact_paths: "experiments/standalone/Bucket/anim*.mp4"
+
```

Comment:
We could also add an extra step to compare CPU and GPU results as we do in ClimaAtmos:

https://github.com/CliMA/ClimaAtmos.jl/blob/fb7b56018491f1d457f708920ca209ecd08b717e/.buildkite/pipeline.yml#L607-L619

(We would probably have to save something that is meaningful to compare)

Reply:
I will save the remapped/interpolated state at the final step. I agree 100% that we should be using the analysis and diagnostics packages you built for ClimaAtmos, but would like to implement that in another PR.